Patch to disable failing test on Aarch64. This is caused by a bug in kim-api 2.4.0 and older.
It was reported upstream and they let me know that they will try to resolve this bug so that we can run this test.
Author: Lara Peeters (UGent)
diff -ruN lammps-stable_29Aug2024_update2.orig/unittest/commands/test_kim_commands.cpp lammps-stable_29Aug2024_update2/unittest/commands/test_kim_commands.cpp
--- lammps-stable_29Aug2024_update2.orig/unittest/commands/test_kim_commands.cpp        2025-03-16 23:02:43.000000000 +0000
+++ lammps-stable_29Aug2024_update2/unittest/commands/test_kim_commands.cpp     2026-02-06 11:29:58.761163107 +0000
@@ -88,7 +88,7 @@
     ASSERT_GE(ifix, 0);
 }

-TEST_F(KimCommandsTest, kim_interactions)
+TEST_F(KimCommandsTest,  DISABLED_kim_interactions)
 {
     if (!LAMMPS::is_installed_pkg("KIM")) GTEST_SKIP();

